PokemonAPI/PokemonAPI/Startup.cs

```diff
@@ -45,6 +47,9 @@ public void ConfigureServices(IServiceCollection services)
             services.ConfigureMVC();
             services.ConfigureAuthentication(Configuration);
             services.ConfigureHealthChecks();
+
+            // configure FluentValidation
+            services.AddMvc().AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<TrainerValidator>());
```

Comment:
Notice that you already have services.ConfigureMVC() on line 47
If you look inside you'll notice the addMVC probably... append this fluent validation there and rename to ConfigureMVCAndFluentValidation